AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -757,7 +757,8 @@ toster.ru###register-banner
 ! https://github.com/AdguardTeam/AdguardFilters/issues/27068
 ||montpellierhandball.com/scripts/fancybox/minified__fancybox.js
 ! https://github.com/AdguardTeam/AdguardFilters/issues/26451
-arcadepunks.com###wpfront-notification-bar
+! https://github.com/AdguardTeam/AdguardFilters/issues/75915
+videokontroldoma.ru,arcadepunks.com###wpfront-notification-bar
```

Comment:
This is the ad leftover, so please make a rule for the russian specific

Reply:
done